src/Components/Card/Card.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+
+function Card({ quizname, numberOfQuestions, url }: { quizname: string, numberOfQuestions: number, url: string }) {
```

Comment:
The types should be extracted out. If the no of parameters in the function increases it will help at that time.